### PR TITLE
Feature/oidf 4

### PR DESCRIPTION
**Pull request by maikel-maas**

### CODE REVIEW

File: modules/openapi/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+An Entity Statement contains the information needed for the Entity that is the subject of the Entity Statement to 
+participate in federation(s). An Entity Statement is a signed JWT. The subject of the JWT is the Entity itself. The 
+issuer of the JWT is the party that issued the Entity Statement. All Entities in a federation publish an Entity Statement 
+about themselves called an Entity Configuration. Superior Entities in a federation publish Entity Statements about their
+Immediate Subordinate Entities called Subordinate Statements.
+
+### Profiles
+
+The Open API generator will generate only models, infrastructures and apis by default. To make it generate apis. To make
```
Comment (nklomp) [inline modules/openapi/README.md:15]: These sentenses stop halfway and are conflicting

File: modules/openapi/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    testImplementation(kotlin("test"))
+}
+
+openApiGenerate {
+    val openApiPackage: String by project
+    generatorName.set("kotlin")
+    packageName.set("com.sphereon.oid.fed.openapi")
+    apiPackage.set("$openApiPackage.api")
+    modelPackage.set("$openApiPackage.models")
+    inputSpec.set("$rootDir/src/main/kotlin/com/sphereon/oid/fed/openapi/openapi.yaml")
```
Comment (jcmelati) [inline modules/openapi/build.gradle.kts:28]: I had to update it to `    inputSpec.set("$rootDir/modules/openapi/src/main/kotlin/com/sphereon/oid/fed/openapi/openapi.yaml")
` for it to run

Comment (nklomp) [reply modules/openapi/build.gradle.kts:28]: Depends on where you are running the command prob

Comment (maikel-maas) [reply modules/openapi/build.gradle.kts:28]: I have the old problem "works on my machine" :smile:

Comment (maikel-maas) [reply modules/openapi/build.gradle.kts:28]: changing from rootDir to projectDir fixed the problem

File: modules/openapi/README.md (hunk 1 above)
Comment (maikel-maas) [reply modules/openapi/README.md:15]: Probabably Ctrl-Z was pressed at some point. Hopefully it makes sense now

**Comment by maikel-maas** [comment]
This PR will be closed and another one will be created for feature/OIDF-33. All comments are already addressed

**Pull request closed without merging**